dask_ml/model_selection/_incremental.py

```diff
@@ -817,6 +818,24 @@ class IncrementalSearchCV(BaseIncrementalSearchCV):
     >>> search = IncrementalSearchCV(model, params, random_state=0,
     ...                              n_initial_parameters=1000,
     ...                              patience=20, max_iter=100)
+
+    ``tol`` and ``patience`` are for stopping training on a "plateau", or when
+    the hold-out score flattens and/or falls. For example, setting ``tol=0``
+    and ``patience=2`` will dictate that scores will always increase until
```

Comment:
We're not dictating that scores will increase (we don't determine that).  We might need to change the language here.